test/integration/blob_container_integration_test.cpp

```diff
@@ -74,17 +74,36 @@ TEST_CASE("Get Container Property", "[container], [blob_service]")
         auto container_name = prefix + as_test::get_random_string(10);
         auto first_outcome = client.create_container(container_name).get();
         REQUIRE(first_outcome.success());
-        auto second_outcome = client.get_container_property(container_name);
+        auto second_outcome = client.get_container_properties(container_name).get();
         REQUIRE(second_outcome.success());
         REQUIRE_FALSE(second_outcome.response().etag.empty());
+
+        std::vector<std::pair<std::string, std::string>> metadata;
+        metadata.emplace_back(std::make_pair("mkey1", "mvalue1"));
+        metadata.emplace_back(std::make_pair("mkEy2", "MValUe2#  %"));
```

Comment:
Ideally, add `%2d` as metadata value to validate as well.

Reply:
Will improve this test case.

Reply:
fixed in https://github.com/Azure/azure-storage-cpplite/pull/50/commits/8daaf0b2617d57ab1b1f50c1595636fbe93dedb3